### PR TITLE
feat(universal-router-sdk): Update for new migration flow

### PR DESCRIPTION
**Pull request by hensha256**
## PR Scope

Please title your PR according to the following types and scopes following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/):

- `fix(SDK name):` will trigger a patch version
- `chore(<type>):` will not trigger any release and should be used for internal repo changes
- `<type>(public):` will trigger a patch version for non-code changes (e.g. README changes)
- `feat(SDK name):` will trigger a minor version
- `feat(breaking):` will trigger a major version for a breaking change

## Description

_[Summary of the change, motivation, and context]_

## How Has This Been Tested?

_[e.g. Manually, E2E tests, unit tests, Storybook]_

## Are there any breaking changes?

_[e.g. Type definitions, API definitions]_

If there are breaking changes, please ensure you bump the major version Bump the major version (by using the title `feat(breaking): ...`), post a notice in #eng-sdks, and explicitly notify all Uniswap Labs consumers of the SDK.

## (Optional) Feedback Focus

_[Specific parts of this PR you'd like feedback on, or that reviewers should pay closer attention to]_

## (Optional) Follow Ups

_[Things that weren't addressed in this PR, ways you plan to build on this work, or other ways this work could be extended]_

### CODE REVIEW

**Comment by dianakocsis** [comment]
https://github.com/Uniswap/sdks/pull/175/files#diff-d88b33523189d4aaedd28fea85995d8d1caf77df2cd50b93d244c1ff050ccd6eL115
no longer multicall

**Review by dianakocsis — COMMENTED** [review]
beautiful!

**Review by dianakocsis — APPROVED** [review]

File: sdks/universal-router-sdk/src/utils/routerCommands.ts [hunk 1]
```diff
@@ -25,7 +25,8 @@ export enum CommandType {
   V4_SWAP = 0x10,
   V3_POSITION_MANAGER_PERMIT = 0x11,
   V3_POSITION_MANAGER_CALL = 0x12,
-  V4_POSITION_CALL = 0x13,
+  V4_INITIALIZE_POOL = 0x13,
+  V4_POSITION_MANAGER_CALL = 0x14,
```
Comment (marktoda) [inline sdks/universal-router-sdk/src/utils/routerCommands.ts:29]: reason for changing command index for posm call? breaking change on the previous version

**Review by marktoda — APPROVED** [review]
lgtm but check my brekaing change comment

File: sdks/universal-router-sdk/src/utils/routerCommands.ts (hunk 1 above)
Comment (hensha256) [reply sdks/universal-router-sdk/src/utils/routerCommands.ts:29]: oh damn didnt see this 🤦‍♀️ mm i dont think there was a a good reason... so probably shouldnt have happened. Thats on me

**Pull request merged**